hubblestack/extmods/returners/splunk_fdg_return.py

```diff
@@ -177,6 +177,14 @@ def _build_args(ret):
 
     return args
 
+def _file_url_to_sourcetype_tag(filename):
```

Comment:
Could this function have a short docstring explaining its purpose? It's clear what it does, it would be a small quality of life thing

Reply:
fair point. pushed patch to that effect